checkpoint.go

```diff
@@ -152,6 +159,11 @@ func Check(p *CheckParams) (*CheckResponse, error) {
 	for flag, value := range p.Flags {
 		v.Set("flag_"+flag, value)
 	}
+	if p.ExtraFlags != nil {
+		for _, d := range p.ExtraFlags() {
+			v.Add("flag_"+d.Key, d.Value)
```

Comment:
Minor suggestion: factorise `"flag_"` into constant, e.g. `KeyPrefix`, to avoid future accidental divergence.